index.js

```diff
@@ -55,7 +55,11 @@ module.exports = function(url, options){
     result.pageTitle = $('head > title').text();
 
     $('[rel=stylesheet]').each(function() {
-      var link = $(this).attr('href');
+        var link = $(this).attr('href');
+        if (!(typeof link !== typeof undefined && link !== false)) {
```

Comment:
Mind pulling in [`is-present`](https://www.npmjs.com/package/is-present) and making the logic `if`/`else`?

```js
if (isPresent(link)) {
  result.links.push(createLink(link, url));
} else {
  result.styles.push(stripHtmlComments($(this).text()));
}
```

Reply:
Yea sure